src/platform/utilities/sso/keepAliveSSO.js

```diff
@@ -43,6 +45,33 @@ export const sanitizeAuthn = authnCtx => {
         .replace(SKIP_DUPE_QUERY.MULTIPLE_QUERIES, emptyString);
 };
 
+export const generateAuthnContext = (
+  { headers } = {
+    headers: {},
+  },
+) => {
+  try {
+    const CSP = headers.get(AUTHN_HEADERS.CSP);
+
+    return {
```

Comment:
since we are not doing anything with the error in the catch, can we just do:

`return CSP ? { ... } : {}`

Reply:
We aren't doing anything with the error, but it's an easier way to check for an empty object if the headers.get fails.  I can change it if you want.